Formula/ecl.rb

```diff
@@ -12,13 +13,23 @@ class Ecl < Formula
     sha256 "f86b66f182afdc3b7a8e8e51ca3e230d6ccc308d760faae1b2b345fd1ead0ab3" => :yosemite
   end
 
+  depends_on "libatomic_ops" => :build
+  depends_on "pkg-config"    => :build
+  depends_on "bdw-gc"
   depends_on "gmp"
+  depends_on "libffi"
```

Comment:
macOS has its own system libffi, which we should use instead of the brew duplicate if possible.